src/env.ts

```diff
@@ -0,0 +1,111 @@
+import dotenv from 'dotenv'
+import * as envalid from 'envalid'
+import { makeValidator } from 'envalid'
+import { singleton } from 'tsyringe'
+
+if (process.env.NODE_ENV === 'test') {
+  dotenv.config({ path: 'tests/test.env' }) //different configs for unit and integration??
+} else {
+  dotenv.config()
+}
+const proofRequestOptions = `{
+    "protocolVersion": "v2",
+    "proofFormats": {
+      "anoncreds": {
+        "name": "drpc-proof-request",
+        "version": "1.0",
+        "requested_attributes": {
+          "companiesHouseNumberExists": {
+            "name": "companiesHouseNumber"
+          }
+        }
+      }
+    }
+  }`
+
+const stringArray = makeValidator((input: string | string[]) => {
+  if (Array.isArray(input)) {
+    return input
+  }
+  if (typeof input !== 'string') {
+    throw new Error('Invalid input type, expected a string')
+  }
+  try {
+    return input.split(',').map((s) => s.trim())
+  } catch (err) {
+    throw new Error('Invalid input for string array')
+  }
+})
+
+const envConfig = {
+  LABEL: envalid.str({ default: 'AFJ Rest', devDefault: 'AFJ Rest Agent' }),
+  WALLET_ID: envalid.str({ default: 'walletId', devDefault: 'walletId' }),
+  WALLET_KEY: envalid.str({ default: 'walletKey', devDefault: 'walletKey' }),
+  ENDPOINT: stringArray({
+    default: ['http://localhost:5002', 'ws://localhost:5003'],
+    devDefault: ['http://localhost:5002', 'ws://localhost:5003'],
+  }),
+  LOG_LEVEL: envalid.num({ default: 3, devDefault: 3 }),
```

Comment:
I thought we'd replaced the logger with `pino`. Should this not be choices like `trace`, `debug`, `info` etc.?